CleanSwift/CleanSwift/NetworkService/NetworkServiceProtocol.swift

```diff
@@ -0,0 +1,14 @@
+//
+//  NetworkServiceProtocol.swift
+//  CleanSwift
+//
+//  Created by Ernest Avagovich on 17.02.2023.
+//
+
+import Foundation
+
+/// Протокол сетевого сервиса
+protocol NetworkServiceProtocol {
+    func getAstronomy(completion: @escaping (Result<PictureOfDay, Error>) -> ())
```

Comment:
```suggestion
    func fetchAstronomy(completion: @escaping (Result<PictureOfDay, Error>) -> ())
```
тебе вроде Женя менял на fetch все get ))